src/EduManagementLab.Api/Controllers/CoursesController.cs

```diff
@@ -231,6 +255,17 @@ public class CourseDto
             [Required]
             public DateTime EndDate { get; set; }
         }
+
+        public class MembershipDto
+        {
+            public Guid Id { get; set; }
+            [Required]
+            public Guid CourseId { get; set; }
+            [Required]
+            public Guid UserId { get; set; }
```

Comment:
Maybe we should return a UserDto instead of the UserId?
If we only return the Id the consumer also has to fetch all users. Right now they have to fetch them one by one from the GET User endpoint.

Reply:
I am not sure about this. Maybe we should discuss it before we add a UserDto.
Maybe it should only include the UserId and the DisplayName.
Or only the Id. We could add a lookup endpoint in the User controller where you pass a list of UserIds to fetch them.

Reply:
Yes we discuss this later.